training/keras_training.py

```diff
@@ -95,6 +95,13 @@ def main(args, config):
     for v in variables:
         logger.debug("%s", v)
 
+    if args.randomization:
+        signal_classes = ["vbftopo_highmjj", "ggh_1J_PTH0to120", "vbftopo_lowmjj", "ggh_1J_PTH120to200", "qqh_2J", "ggh_0J", "ggh_PTHGT200", "qqh_PTHGT200", "ggh_2J", "ggh", "qqh"]
```

Comment:
~~~
signal_classes = [class_ for class_ in  classes if class_.startswith(('ggh','qqh','vbftopo'))]
~~~

Reply:
Sure, why not